nupic/engine/__init__.py

```diff
@@ -729,6 +729,13 @@ def inspect(self):
     from nupic.analysis import inspect    
     inspect(self)
 
+  @staticmethod
+  def addCustomRegionPackage(path):
```

Comment:
Perhaps rename to something like registerRegionPackage or registerCustomRegionPackage. There are other functions in this file that start with "add" like addRegion, etc that this could be confused with otherwise. And I'd drop "custom" since you would only be registering a region package if it was custom so seems redundant but will leave it up to you.